### PR TITLE
DB start

### PR DESCRIPTION
**Pull request by mazhara**
- Set a target: We're going all into web development! Plus DBs. 
  - Choose libs for DB: postgresql / skunk, UI: Tyrian
  - Set up testcontainers: add integration module, check if postgres image is up and running 
  - Wrote a test with ScalaTest.
  - The testcontainers docs are out of date.
  - Learned how to troubleshoot docker containers:
    - Looking at Docker desktop to see running containers.

### CODE REVIEW

**Review by kannupriyakalra — APPROVED** [review]
LGTM

**Comment by noelwelsh** [comment]
Please run the `prePR` command in sbt and push the resulting changes. CI will fail less after this.

**Comment by assayire** [comment]
I am not able to see this branch after a git pull. Anybody else seeing this problem? I am wondering if it is because of the branch naming, which has a `:`.

**Comment by noelwelsh** [comment]
> I am not able to see this branch after a git pull. Anybody else seeing this problem? I am wondering if it is because of the branch naming, which has a `:`.

Probably because the PR originates from a different repository: https://github.com/mazhara/SnorriScalason/tree/db-and-containers

File: integration/src/test/scala/Foo.scala [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (assayire) [inline integration/src/test/scala/Foo.scala]: Can you delete this file?

**Comment by noelwelsh** [comment]
I'm merging now the unnecessary file has been deleted.

**Pull request merged**